easybuild/easyconfigs/i/IPM/darshan-mk-log-dirs.pl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/perl -w
```

Comment:
The question remains: why are you adding this file here? This repository should contain only build recipes and patches, source files should come from elsewhere and the recipes here should only refer to files external to this repository.